engine/src/main/java/org/terasology/rendering/dag/stateChanges/SetViewportSizeOf.java

```diff
@@ -27,35 +27,33 @@
  */
 public final class SetViewportSizeOf implements FBOManagerSubscriber, StateChange {
     private static final String DEFAULT_FBO = "sceneOpaque";
-    private static SetViewportSizeOf defaultInstance;
+    private static SetViewportSizeOf defaultInstance = new SetViewportSizeOf(DEFAULT_FBO);
+    private static FrameBuffersManager frameBuffersManager;
     private SetViewportSizeOfTask task;
 
     private FBO fbo;
     private String fboName;
-    private FrameBuffersManager frameBuffersManager;
 
-    public SetViewportSizeOf(String fboName, FrameBuffersManager frameBuffersManager) {
+    public SetViewportSizeOf(String fboName) {
         this.fboName = fboName;
-        this.frameBuffersManager = frameBuffersManager;
-        fbo = frameBuffersManager.getFBO(fboName);
+        if (!fboName.equals(DEFAULT_FBO)) {
+            fbo = frameBuffersManager.getFBO(fboName);
+        }
```

Comment:
I'm not sure about this IF block. Why do we need it? Can we just store only the fboName and then get the fbo itself only when we need it?